test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1920,6 +1931,22 @@ def test_representation_maxpool2d(self):
             non_ref_node_occurrence={}
         )
 
+    def test_representation_adaptive_avg_pool2d(self):
+        quantizer = XNNPACKQuantizer()
+        operator_config = get_symmetric_quantization_config(is_per_channel=True)
+        quantizer.set_global(operator_config)
+        m_eager = TestHelperModules.ConvWithAdaptiveAvgPool2d().eval()
+
+        example_inputs = (torch.randn(1, 3, 3, 3),)
+
+        self._test_representation(
```

Comment:
My concern with all the rewrites is testing. I remember you said dynamo export is timing out. Do you know the issue? Else we are landing stuff that may not be functionally correct

Reply:
I have tested these locally, yeah I plan to have a repro soon